common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Location.scala

```diff
@@ -10,6 +10,7 @@ case class DigitalLocation(
   license: Option[License] = None,
   credit: Option[String] = None,
   accessConditions: List[AccessCondition] = Nil,
+  imageSourceIds: List[SourceIdentifier] = Nil,
```

Comment:
After some discussion with @warrd, decided not to make this a (parametrised) `Id` type or a wrapped type that implemented `HasId`, given that it will always be `Identifiable` and that this is not the same entity that will later become a fully-formed `Image`